README.md

```diff
@@ -4,77 +4,66 @@
 
 ## About
 
-The SpatioTemporal Asset Catalog (STAC) specification aims to standardize the way geospatial assets are exposed online and queried. 
-A 'spatiotemporal asset' is any file that represents information about the earth captured in a certain space and 
-time. The initial focus is primarily remotely-sensed imagery (from satellites, but also planes, drones, balloons, etc), but 
-the core is designed to be extensible to SAR, full motion video, point clouds, hyperspectral, LiDAR and derived data like
-NDVI, Digital Elevation Models, mosaics, etc. 
-
-The goal is for all major providers of imagery and other earth observation data to expose their data as SpatioTemporal Asset 
-Catalogs, so that new code doesn't need to be written whenever a new JSON-based REST API comes out that makes its data 
-available in a slightly different way. This will enable standard library components in many languages. STAC can also be
-implemented in a completely 'static' manner, enabling data publishers to expose their data by simply publishing linked JSON
-files online.
+The SpatioTemporal Asset Catalog (STAC) family of specifications aims to standardize the way
+geospatial asset metadata is structured and queried. A "spatiotemporal asset" is any file 
+that represents information about the Earth at a certain place and time. The original focus was on scenes of satellite
+imagery, but now covers a broad variety of uses, including sources such as aircraft and drone, and data such as hyperspectral optical, synthetic aperture radar (SAR), video, point clouds, lidar, digital elevation models (DEM), vector, machine learning labels, and composites like NDVI and mosaics. STAC is intentionally designed with a minimal core and flexible extension semantics to support a broad set of use cases.
+
+STAC defines a common format (object types) and interface (web service API) to be used by providers and consumers of geospatial data.  This is advantageous to providers, as they can simply use a well-designed, standard format without needing to design their own proprietary one. This is advantageous to consumer because they can use existing libraries and tools to access metadata, instead of needing to write new code to interact with each data provider's proprietary formats and APIs. 
```

Comment:
I think we need to 'introduce' the web service API, explaining it's part of the STAC 'family', but not in this repo, and link to the stac-api-spec repo. May make sense to do that a bit lower, and just focus on the common format here.

Also small typo consumer -> consumers

(Also, I do recommend keeping lines in markdown to 80/100 characters or less, as it makes commenting and suggesting changes way easier, as they can be more granular - this line would have 5 different spots to enter a comment, instead of just one, and I could have just done two separate comments, one commitable)